content/blog/20240430_fluentd-v1.17.0-has-been-released.md

```diff
@@ -0,0 +1,49 @@
+# Fluentd v1.17.0 has been released
+
+Hi users!
+
+We have released v1.17.0 on 2024-04-30. ChangeLog is [here](https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v1170---20240430).
+
+This release is a new release of v1.17 series.
+In this release, we added some new features for some plugins, and fixed bugs of Parser.
```

Comment:
```suggestion
In this release, we added some new features for some plugins and fixed bugs of Parser.
```

You don't need to use comma here. Because there is no precedent listed items here.
`some new features for some plugins` is just one item.